pom.xml

```diff
@@ -553,7 +553,7 @@ under the License.
 			<dependency>
 				<groupId>org.apache.commons</groupId>
 				<artifactId>commons-lang3</artifactId>
-				<version>3.12.0</version>
+				<version>3.18.0</version>
```

Comment:
nit: it would be better to have this declared as a variable at the top of the pom file.